### PR TITLE
Pd 21880 subscription api enhancement

### PR DESCRIPTION
**Pull request by salmanHcart**
Adding New Objects to  subscription api

### CODE REVIEW

File: acapi.yaml [hunk 1]
```diff
@@ -11776,6 +11776,289 @@ components:
...
+                configuration: ''
+                variants: null
+                personalizations: null
+            lookup_key: Tr0anQUb2yM%3D
+          customer:
+            id: 1
+            customer_number: ''
+            last_name: Barlow
+            first_name: Stefan
+            email: stefan@americomlp.com
```
Comment (awhite-cart) [inline acapi.yaml:11862]: Please remove any real information in these examples.

File: acapi.yaml [hunk 2]
```diff
@@ -11776,6 +11776,289 @@ components:
...
+            sales_person_user_id: null
+            store_payment_methods_enabled: []
+            tax_rate: null
+            reward_tier_id: null
+            sub: ''
+          shipping_address:
+            id: 267
+            customer_id: 1
+            company: Cart
+            address_line_1: mckldmcdm
```
Comment (awhite-cart) [inline acapi.yaml:12035]: Can you change any referecence of 'mckldmcdm' to '123 Test' or something similar?

File: acapi.yaml (hunk 1 above)
Comment (salmanHcart) [reply acapi.yaml:11862]: Sure done

File: acapi.yaml (hunk 2 above)
Comment (salmanHcart) [reply acapi.yaml:12035]: yes Sure, Done

File: acapi.yaml [hunk 3]
```diff
@@ -11776,6 +11776,289 @@ components:
...
+            id: 98
+            payment_method_id: 193
+            customer_id: 8
+            payment_type: CreditCard
+            is_default: true
+            card_id: 33
+            updated_at: '2022-03-07T21:57:16.773+05:00'
+            created_at: '2022-03-07T21:57:16.773+05:00'
+            address_id: null
+      description: If you want child resources included in response you must pass the query parameter 'includeLookups = true'
```
[the remark this answers is not in the document]
Comment (salmanHcart) [reply acapi.yaml:12060]: image

[the remark this answers is not in the document]
Comment (salmanHcart) [reply acapi.yaml:12060]: below Heading in Description

**Pull request merged**